src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerCallableStatement.java

```diff
@@ -17,20 +17,6 @@
  *
  */
 public interface ISQLServerCallableStatement extends java.sql.CallableStatement, ISQLServerPreparedStatement {
-    /**
-     * Sets parameter parameterName to DateTimeOffset x
-     * 
-     * @param parameterName
-     *            the name of the parameter
-     * @param x
-     *            DateTimeOffset value
-     * @throws SQLException
-     *             if parameterName does not correspond to a named parameter; if the driver can detect that a data conversion error could occur; if a
-     *             database access error occurs or this method is called on a closed <code>CallableStatement</code>
-     */
-    public void setDateTimeOffset(String parameterName,
```

Comment:
This should be kept in the interface, as it isn't part of JDBC Specs for [Prepared Statement](https://docs.oracle.com/javase/9/docs/api/java/sql/PreparedStatement.html)